demisto_sdk/commands/content_graph/parsers/related_files.py

```diff
@@ -146,6 +148,23 @@ class SchemaRelatedFile(RelatedFile):
     def get_optional_paths(self) -> List[Path]:
         return [Path(str(self.main_file_path).replace(".yml", "_schema.json"))]
 
+    @cached_property
+    def file_content(self) -> Optional[Dict[str, Any]]:
+        """
+        Reads and returns JSON content from the first optional path.
+        Returns None if the file cannot be read or parsed.
+        """
+        paths = self.get_optional_paths()
+        if not paths:
+            return None  # No paths available
+        try:
+            with open(paths[0], "r") as file:
```

Comment:
in TextFiles.file_content we separate to cases: whether git_sha exists, or otherwise. do we need to do it here as well?